src/chanjo2/crud/intervals.py

```diff
@@ -56,7 +65,7 @@ def _filter_intervals_by_hgnc_symbols(
     interval_type: Union[SQLGene, SQLTranscript, SQLExon],
     hgnc_symbols: List[str],
 ) -> List[Union[SQLGene, SQLTranscript, SQLExon]]:
```

Comment:
```suggestion
) -> query.Query:
```
I do not think you are reading the db yet and interpolating to real objects